lib/src/data/repositories/database_repository.dart

```diff
@@ -17,14 +14,7 @@ class DatabaseRepository implements IDatabaseRepository {
     MovieEntity movie,
     EndpointEnum endpoint,
   ) async {
-    MovieEntity? existingMovie = await getMovieById(movie);
-    if (existingMovie != null &&
-        !existingMovie.category.contains(endpoint.name)) {
-      existingMovie.category.add(endpoint.name);
-      await database.movieDao.insertMovie(existingMovie);
-    } else {
-      await database.movieDao.insertMovie(movie);
-    }
```

Comment:
What happened to this? Where is this processed now?

Reply:
That method became obsolete as it was not necessary, it was only used when inserting a new movie, but now I let the '@Insert(onConflict: OnConflictStrategy.replace)' in the DAO handle this.